src/igvfd/schemas/changelogs/analysis_set.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`analysis_set.json`*
 
+### Minor changes since schema version 8
+
+* Add external_image_data_url
```

Comment:
The new property should be enclosed by the `` characters, also with a period at the end. Add `external_image_data_url`.